### PR TITLE
resprite and add lore to cryo cell

### PR DESCRIPTION
**Pull request by novaepee**
## About The Pull Request

Welcome to the ExoDyne Pharmaceutics cryogentic restoration cell customer package.

This is a demonstration of how to use the ExoDyne Pharmaceutics cryogentic restoration cell in order to treat patients with trauma.

https://github.com/tgstation/TerraGov-Marine-Corps/assets/6610922/5123330a-3d14-40e3-b505-c7d656c5d731

## Why It's Good For The Game

This sprite was made by Azlan, and I modified it to fit our need.

I saw Azlan's cryo cell, and I went, "Hmm! We should use this to distinguish the vat born cell and cryo cell", then I added lore because why not.

## Changelog

:cl: novaepee, Azlan
add: resprite and add lore to cryo cell
/:cl:

### CODE REVIEW

**Comment by coldironwarrior** [comment]
im a fan; love it as a unique take on cryogenics versus Big Green Tube of Goop

**Comment by Arctrooper8** [comment]
New sprite is really nice, and I'm a big sucker for little bits of lore and worldbuilding that are scattered in item and object descriptions. I especially like the usage of Exodyne.

**Comment by FosterGrime-Heymynameslouie** [comment]
THE FREEZER

**Comment by XElectricX** [comment]
I think it looks out of place, too modern. Great sprite though so hard to say no to it. Maybe make that the vat birthing chamber instead?

**Comment by Lumipharon** [comment]
I think the tube is better tbh

**Comment by TiviPlus** [comment]
Crunch tube has soul

File: code/modules/atmospherics/machinery/components/unary_devices/cryo.dm [hunk 1]
```diff
@@ -1,7 +1,8 @@
 #define CRYOMOBS 'icons/obj/cryo_mobs.dmi'
 
 /obj/machinery/atmospherics/components/unary/cryo_cell
-	name = "cryo cell"
+	name = "ExoDyne Pharmaceutics cryogentic restoration cell"
```
[the remark this answers is not in the document]
Comment (novaepee) [reply code/modules/atmospherics/machinery/components/unary_devices/cryo.dm:4]: corpo agents say that this is important

**Comment by Lumipharon** [comment]
Closing as this is also a walance issue.  This new sprite is huge and completely hide the turf above the tube.
Also ugo.

**Pull request closed without merging**